exercises/strings/countWords/countWords.js

```diff
@@ -8,16 +8,14 @@
  */
 
 function countWords(string) {
-  let array =[];
-  for (x of string){
-    if (x === ' '){
-      array.push(1)
-    }
-    console.log(array)
-    return array.length +1
+  let n = 0;
+  for (let i=0; i <= string.length;i += 1) {
```

Comment:
I see that you are trying to count the words by incrementing a counter (n) whenever there is a space in the word.  This is an interesting approach but may not always be reliable.  A way to simplify this may be to see if there is a JavaScript function that can convert a string of words into an array to make counting the words in the sentence easier